### DISCUSSION TITLE
Notice type "change" not allowed for subtype 14

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#546] [Q&A] [answered]
When testing to create a change notice for a subtype 14, we got the following error: Error BR-BT-00002-0100, field BT-02 (notice type)

In the Schematron rule for Error BR-BT-00002-0100, field BT-02 (notice type), it says that for Legal basis "32014L0023", only the following notice types are allowed: 'pin-cfc-social','cn-standard','veat','can-standard','can-social'. Notice type "change" is not allowed. Will this be corrected for the rule or is there something we have misunderstood regarding change notices?
(We have only discovered this for subtype 14 so far, so it might apply to other subtypes if a correction is needed).

Kind regards,
Åsa

**Comment by ferrakaop** [comment, marked as answer]
Starting with SDK 1.6.0 you can and should use the same notice type as the notice you are changing, e.g. a "cn-standard" remains a "cn-standard" when it's changed - it just has an additional Change section.  This applies to all forms.

We intend to remove "change" notice type (and "corr" form type) fully in SDK 1.9.

**Comment by rengit62** [comment]
Hi Karl, just a question about change notice, taking for example notice type 29
field BT-140-notice is read-only true . is it  an error or is it 'wanted'  ? And in the second choice which value should we put ?
Thanks and Regards
Renato

**Comment by ferrakaop** [comment]
eNotices2 handles Change notices with a specific pop-up to enter the reason before updating the fields on the screen. So BT-140 is kept read-only to ensure consistency. But you can adapt this part of the SDK to your needs if you wish.
